src/bin/ci-generate.rs

```diff
@@ -126,8 +126,8 @@ impl<T: Serialize> Provider for ClapSerialized<T> {
 #[derive(Parser, Debug, Serialize, Deserialize)]
 struct CargoData {
     /// Docker image description.
-    #[clap(long)]
-    docker_image_description: String,
+    #[clap(long, default_value = "")]
```

Comment:
Can we remove `default_value` here? It should automatically set `None` as default value, since the type is an option